serde_at/src/de/mod.rs

```diff
@@ -737,8 +806,23 @@ mod tests {
         );
     }
 
+    #[test]
+    fn char_test() {
+        assert_eq!(crate::from_str("+CCID: B"), Ok(CharHandle('B')));
+    }
+
     #[test]
     fn newtype_struct() {
         assert_eq!(crate::from_str("+CCID: 15"), Ok(Handle(15)));
     }
+
+    #[test]
+    fn char_vec_struct() {
+        #[derive(Debug, Deserialize, PartialEq)]
+        struct CharVecTest(CharVec<consts::U4>);
+        let stru = CharVecTest(CharVec(
+            heapless::Vec::from_slice(&['I', 'M', 'P', '_']).unwrap(),
+        ));
+        assert_eq!(crate::from_str("+CCID: IMP_"), Ok(stru));
```

Comment:
No reason for the extra wrapper here?

```suggestion
        let expectation = CharVec(
            heapless::Vec::from_slice(&['I', 'M', 'P', '_']).unwrap(),
        );
        assert_eq!(crate::from_str("+CCID: IMP_"), Ok(expectation));
```